scenarioo-client/app/build/scenarios-overview/scenarios-overview.component.html

```diff
@@ -15,133 +15,125 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<div class="row sc-space-bottom">
-    <div [ngClass]="isPanelCollapsed ? 'sc-space-top col-sm-11' : 'sc-space-top col-sm-8'">
-        <div id="left">
-            <sc-title useCaseName="{{useCaseName | scHumanReadable}}"></sc-title>
-        </div>
-        <div id="right">
-            <label class="input-group">
-                <input [(ngModel)]="searchTerm" class="form-control" data-toggle="tooltip"
-                       id="scenariosSearchField"
-                       placeholder="Search in use case &quot;{{useCaseName | scHumanReadable}}&quot;..."
-                       title="type words separated by spaces to search for" type="text" autofocus/>
-                <span class="input-group-btn">
+<sc-detailarea [mainDetailsSections]="mainDetailsSections" [additionalDetailsSections]="additionalDetailsSections" key="useCaseView">
+
+    <sc-title useCaseName="{{useCaseName | scHumanReadable}}"></sc-title>
+    <div class="searchbar">
+        <label class="input-group">
+            <input [(ngModel)]="searchTerm" class="form-control" data-toggle="tooltip"
+                   id="scenariosSearchField"
+                   placeholder="Search in use case &quot;{{useCaseName | scHumanReadable}}&quot;..."
+                   title="type words separated by spaces to search for" type="text" autofocus/>
+            <span class="input-group-btn">
                     <button (click)="resetSearchField()" class="btn btn-default" type="button">reset</button>
-                </span>
-            </label>
-        </div>
+            </span>
+        </label>
     </div>
 
-    <div id="sc-meta-data-panel">
-        <sc-detailarea (togglePannelCollapsedValue)="collapsePanel($event)" [mainDetailsSections]="mainDetailsSections" [additionalDetailsSections]="additionalDetailsSections" key="useCaseView"></sc-detailarea>
+    <p class="tableTitle">
```

Comment:
I would use div instead of `<p>` elements - or what is the reason for the `<p>` elem here??

Reply:
done